changelogs/current.yaml

```diff
@@ -142,6 +142,11 @@ removed_config_or_runtime:
     removed ``envoy.reloadable_features.do_not_await_headers_on_upstream_timeout_to_emit_stats`` and legacy code paths.
 
 new_features:
+- area: grpc_json_transcoder
+  change: |
+    added ``max_message_size`` field, which can either increase or decrease the size of messages that
```

Comment:
```suggestion
    added ``max_body_size`` field, which can either increase or decrease the size of messages that
```

Reply:
Done (but now `max_request_body_size` and `max_response_body_size` per qiwzhang's request).